sdk/python/jobs/automl-standalone-jobs/automl-classification-task-bankmarketing/automl-classification-task-bankmarketing.ipynb

```diff
@@ -13,21 +13,17 @@
     "- A Compute Cluster. [Check this notebook to create a compute cluster](../../../resources/compute/compute.ipynb)\n",
     "- A python environment\n",
     "- Installed Azure Machine Learning Python SDK v2 - [install instructions](../../../README.md) - check the getting started section\n",
-    "- The latest MLFlow packages:\n",
     "\n",
```

Comment:
Could you please check with @rjaincc, @novaturient95 on the list of best practices we have captured in vision and potentially add them here if we are missing anything?

Reply:
For mlflow installation, that's also what we have. @skasturi - were you referring to something else, apart from this?